# Current profile to use (0-4) 
#Please set the values in the wheelchair as multiple of 5 for correct operation
current_profile: 0

# Speed profiles
profiles:
  profile0:
    max_forward_level: 25    # Maximum forward speed level (5-100)
    min_forward_level: 10     # Minimum forward speed level (5-100)
    max_backward_level: 10    # Maximum backward speed level (5-100)
    min_backward_level: 10    # Minimum backward speed level (5-100)
    max_turning_level: 10     # Maximum turning speed level (5-100)
    min_turning_level: 10     # Minimum turning speed level (5-100)

  profile1:
    max_forward_level: 40
    min_forward_level: 15
    max_backward_level: 15
    min_backward_level: 15
    max_turning_level: 15
    min_turning_level: 15

  profile2:
    max_forward_level: 60
    min_forward_level: 25
    max_backward_level: 20
    min_backward_level: 20
    max_turning_level: 20
    min_turning_level: 20

  profile3:
    max_forward_level: 100
    min_forward_level: 25
    max_backward_level: 25
    min_backward_level: 20
    max_turning_level: 20
    min_turning_level: 20

  profile4:
    max_forward_level: 100
    min_forward_level: 35
    max_backward_level: 30
    min_backward_level: 20
    max_turning_level: 20
    min_turning_level: 20
